data_validation/cli_tools.py

```diff
@@ -146,15 +145,36 @@ def get_parsed_args() -> Namespace:
     """Return ArgParser with configured CLI arguments."""
     parser = configure_arg_parser()
     args = ["--help"] if len(sys.argv) == 1 else None
-    return parser.parse_args(args)
+    parsed_args = apply_post_parse_checks(parser.parse_args(args), parser)
+    return parsed_args
+
+
+def apply_post_parse_checks(args: Namespace, parser) -> Namespace:
```

Comment:
Instead of doing this, maybe we should split out custom query row versus column into separate subparsers like we do for data-validation validate (row | column). This way we can have the correct set of flags for each type of validation. We can also require PKs for `data-validation validate custom-query row` and support hash/concat/comp-fields only for row.

This would be a breaking change so we should add the exclamation point 'fix!' to the PR title as per the [conventional commits guidelines](https://www.conventionalcommits.org/en/v1.0.0/), and update our documentation to support `data-validation validate custom-query (row|column)`

Reply:
Done!
Added a subparser for `data-validation validate custom-query` to separate `row` and `column`.
Removed `--primary_keys` arg for `validate column` and `validate custom-query column`